samples/tfm_integration/psa_level_1/prj.conf

```diff
@@ -17,3 +17,6 @@ CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
 
 CONFIG_MAIN_STACK_SIZE=4096
 CONFIG_HEAP_MEM_POOL_SIZE=4096
+
+# Set adequate TFM isolation level
+CONFIG_TFM_ISOLATION_LEVEL=1
```

Comment:
I think this is redundant (the default is 1, anways), but also it might overlap with other platforms (e.g. lpcxpresso55s69) that select profile MEDIUM, which enabled Isolation Level 2.
@microbuilder what is your opinion?
@erwango is the setting of TFM isolation here strictly required?

Reply:
@ioannisg I don't have strong feelings about it, and I don't mind moving it back to the original commit. I regret the name of this sample, to be honest, since it isn't really specific to level 1 ... I named it as a demo when working on PSA Level 1 certification for Zephyr showing how to check some of the boxes off like key management.

Any suggestions on a better name, removing the `level_1` part since it isn't actually a requirement to be level 1?

Reply:
FYI, using level 2 isolation is rejected for by TFM:
```
CMake Error at config/check_config.cmake:18 (message):
  INVALID CONFIG: TFM_ISOLATION_LEVEL > 1 AND NOT TFM_PSA_API
```
I've not gone deeper in the analysis. I thought you guys should know :-)

Reply:
@erwango Sorry, lets move this back to a board-specific value for now, and I plan on reworking this sample a bit anyway and can update isolation level checks in another PR.

Reply:
No problem!

Reply:
All I am saying is that if we build for the lpcxpresso55s69, which uses profile medium (@microbuilder could you check why it is like that?) we enable both TFM_PSA_API and TFM_ISOLATION_LEVEL=2, so there's a conflict here.

Reply:
I'm not sure why that platform is defaulting to medium off the top of my head, but I suspect it can and should be set to small, but I'll investigate.

Reply:
Cool! Let's take this out of scope of this PR (unless we see CI issues).